airbyte-webapp/jest.config.ts

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line jest/no-jest-import
+import type { Config } from "jest";
```

Comment:
:information_source: Actual jest config, compiled from our previous `package.json` plus some additional config we need now, since CRA doesn't take care of it anymore.